shelley/chain-and-ledger/formal-spec/Properties.md

```diff
@@ -256,13 +292,19 @@ the epoch nonce is what you get from combining the blocks leading up to it
 The overlay schedule is obeyed: no blocks are produced during the silent blocks,
 and only core nodes makes blocks on the overlay slots.
 
+**Property** Full decentralization is reached eventually
```

Comment:
I like the idea of this one, but I think in practices it just becomes "we can vote d=0" as a protocol parameter.under a given rule I wonder what we could add to this to make the tests more meaningful...

Reply:
i was going to change it to the tautology "full decentralization can be achieved", if thats any more useful. But I am also wondering about whether there should be any discussion (properties?) about going backwards towards centralization